gears/src/main/scala/nl/lumc/sasc/biopet/pipelines/gears/GearsReport.scala

```diff
@@ -41,120 +52,493 @@ object GearsReport extends MultisampleReportBuilder {
+            List(
+              "Non-unique" -> Future.successful(
+                ReportPage(
+                  List(),
+                  List("All mappings" -> ReportSection(
+                    "/nl/lumc/sasc/biopet/pipelines/gears/krakenKrona.ssp",
+                    centrifugePageAllMappingsKronaPlot
+                  )),
+                  Map()
+                ))),
+            List("Unique mappings" -> ReportSection(
+              "/nl/lumc/sasc/biopet/pipelines/gears/krakenKrona.ssp",
+              centrifugePageUniqueMappingsKronaPlot)),
+            Map("summaryModuleTag" -> "gearscentrifuge", "centrifugeTag" -> Some("centrifuge"))
+          )))
+      else None
+    val krakenPageKronaPlot = GearsKronaPlot
+      .values(summary, runId, "gearskraken", "krakenreport", samples, libraries, sampleId, libId)
+    val krakenAnalysisPage =
+      if (krakenExecuted)
+        Some(
+          "Kraken analysis" -> Future.successful(
+            ReportPage(
+              List(),
+              List("Krona plot" -> ReportSection(
+                "/nl/lumc/sasc/biopet/pipelines/gears/krakenKrona.ssp",
+                krakenPageKronaPlot)),
+              Map()
+            )))
+      else None
+
+    val qiimeClosesOtuTablePage =
+      if (qiimeClosesOtuTable.isDefined)
+        Some(
+          "Qiime closed reference analysis" -> Future.successful(ReportPage(
+            List(),
+            List("Krona plot" -> ReportSection(
+              "/nl/lumc/sasc/biopet/pipelines/gears/qiimeKrona.ssp")),
+            Map("biomFile" -> new File(qiimeClosesOtuTable.get.path))
+          )))
+      else None
+
+    val qiimeOpenOtuTablePage =
+      if (qiimeOpenOtuTable.isDefined)
+        Some(
+          "Qiime open reference analysis" -> Future.successful(
+            ReportPage(List(),
+                       List("Krona plot" -> ReportSection(
+                         "/nl/lumc/sasc/biopet/pipelines/gears/qiimeKrona.ssp")),
+                       Map("biomFile" -> new File(qiimeOpenOtuTable.get.path)))))
+      else None
+    val flexiprepReadSummary =
+      FlexiprepReadSummary.values(summary, runId, samples, libraries, sampleId)
+    val flexiprepBaseSummary =
+      FlexiprepBaseSummary.values(summary, runId, samples, libraries, sampleId)
+    Future {
+      ReportPage(
+        List(flexiprepReportPage,
+             centrifugeReportPage,
+             krakenAnalysisPage,
+             qiimeClosesOtuTablePage,
+             qiimeOpenOtuTablePage).flatten,
+        List(
+          "QC reads" -> ReportSection(
+            "/nl/lumc/sasc/biopet/pipelines/flexiprep/flexiprepReadSummary.ssp",
+            flexiprepReadSummary),
+          "QC bases" -> ReportSection(
+            "/nl/lumc/sasc/biopet/pipelines/flexiprep/flexiprepBaseSummary.ssp",
+            flexiprepBaseSummary)
+        ),
+        args
+      )
+    }
   }
 
-}
\ No newline at end of file
+}
+
+object GearsKronaPlot {
```

Comment:
The size of this file did grow a lot so maybe it now makes more sense to move this object to it's own scala file